Mathlib/CategoryTheory/Limits/Preserves/Basic.lean

```diff
@@ -694,6 +694,28 @@ lemma reflectsColimits_of_reflectsIsomorphisms {G : C ⥤ D} [G.ReflectsIsomorph
 
 end
 
+section
+
+open Functor
+
+variable {C D J : Type*} [Category C] [Category D] [Category J]
+variable (K : J ⥤ C) (L L' : C ⥤ D) (α : L ⟶ L') [IsIso (whiskerLeft K α)]
+variable (c : Cocone K) (hc : IsColimit c) [PreservesColimit K L] [PreservesColimit K L']
+
+include hc in
+lemma isIso_of_colimit_of_natIso : IsIso (α.app c.pt) := by
+  obtain ⟨hc₁⟩ := PreservesColimit.preserves (F := L) hc
```

Comment:
The `PreservesColimit` API should be used using terms like `isColimitOfPreserves L hc`.